Helper/Order.php

```diff
@@ -430,7 +429,9 @@ private function createOrder($quote, $transaction, $frontend)
         // Emulate frontend area in order for email
         // template to be loaded from the correct path
         // even if run from the hook.
-        $this->appState->emulateAreaCode('frontend', function () use ($order){
+        $areaCode = $frontend ? 'frontend' : \Magento\Backend\App\Area\FrontNameResolver::AREA_CODE;
```

Comment:
I think this contradicts with comment above:

```
// Send order confirmation email to customer.
// Emulate frontend area in order for email
// template to be loaded from the correct path
// even if run from the hook.
```

Reply:
Yes, I will delete this comment.